src/service/ActivityService.ts

```diff
@@ -11,9 +11,9 @@ export class ActivityService {
   public static _name = "Activity"
   public static Router = Router()
 
-  public static async list(auth: any, study_id: string, ignore_binary: boolean, sibling: boolean = false) {
+  public static async list(auth: any, study_id: string, ignore_binary: boolean, sibling = false) {
```

Comment:
Please **DO NOT** remove typing information.

Reply:
Okay. Actually linting remove it

Reply:
That is odd and unfortunate as it should not be doing that. If the linter changed it then for now it's fine but let's adjust the `tsconfig.json` to be as **strict as possible** and adjust the eslint/prettier config in `package.json` to be strict as possible as well without deleting typing info.

Reply:
Okay. I have pushed the changes